internal/cephfs/core/clone_test.go

```diff
@@ -17,13 +17,12 @@ limitations under the License.
 package core
 
 import (
-	"errors"
 	"testing"
 
 	cerrors "github.com/ceph/ceph-csi/internal/cephfs/errors"
 
 	fsa "github.com/ceph/go-ceph/cephfs/admin"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
not: no need to change to `require`, could have stayed `assert` as that also provides `ErrorIs()` and there are is no dependent usage of the result. Not sure what golangci-lint reported about this though.

Reply:
can `github.com/stretchr/testify/assert` be removed from `go.mod` and `vendor/` after these changes?

Reply:
i assume if its not required, CI would have caught it, will check and address it if required.

Reply:
Thanks, I'm merely wondering about it.

Reply:
```
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f ceph-csi $]go mod tidy
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f ceph-csi $]git diff
```
i did run `go mod tidy` didnt find anything as we have `github.com/stretchr/testify` as import in go.mod